sources/mongodb_pipeline.py

```diff
@@ -0,0 +1,109 @@
+from typing import List
+
+import dlt
+from dlt.common import pendulum
+from dlt.pipeline.pipeline import Pipeline
+from dlt.common.pipeline import LoadInfo
+
+try:
+    from .mongodb import mongodb, mongodb_collection  # type: ignore
+except ImportError:
+    from mongodb import mongodb, mongodb_collection
+
+
+def load_select_collection_db(pipeline: Pipeline = None) -> LoadInfo:
+    """Use the mongodb source to reflect an entire database schema and load select tables from it.
+
+    This example sources data from a sample mongo database data from [mongodb-sample-dataset](https://github.com/neelabalan/mongodb-sample-dataset).
+    """
+    if pipeline is None:
+        # Create a pipeline
+        pipeline = dlt.pipeline(
+            pipeline_name="pipeline1", destination="duckdb", dataset_name="mongo_data"
```

Comment:
maybe you can name pipeline as in docs?

Reply:
you can use the same pipeline name everywhere

but set the dataset name so it corresponds the data being loaded (ie. collections, databases)